### PR TITLE
Update requirements

### PR DESCRIPTION
**Pull request by ethanhs**

### CODE REVIEW

File: requirements-minimal.txt [hunk 1]
```diff
@@ -1,5 +1,6 @@
+cryptography<=3.3.2
 emoji
 ocflib
 pyOpenSSL
 slackclient<2
-Twisted
+Twisted<=20.3.0
```
Comment (jvperrin) [inline requirements-minimal.txt:6]: Why did you need to restrict Twisted?

Could you add a comment for that and for the cryptography restriction (which I assume is related to the same `rustc` stuff mentioned in ocf/ircbot#193)

**Review by jvperrin — APPROVED** [review]
Seems reasonable, just one question

File: requirements-minimal.txt (hunk 1 above)
Comment (ethanhs) [reply requirements-minimal.txt:6]: Ah yeah I should leave comments. Twisted added incomplete types which messes with mypy, so I limited to an earlier release.

**Review by jvperrin — APPROVED** [review]
